modules/deploy/pages/redpanda/manual/resilience/shadowing-guide.adoc

```diff
@@ -0,0 +1,208 @@
+= Shadowing Guide
+:description: Step-by-step emergency guide for failing over Redpanda shadow links during disasters.
+:env-linux: true
+:page-categories: Management, High Availability, Disaster Recovery, Emergency Response
+
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+
+This guide provides step-by-step procedures for emergency failover when your primary Redpanda cluster becomes unavailable. Follow these procedures only during active disasters when immediate failover is required.
+
+[IMPORTANT]
+====
+This is an emergency procedure. For planned failover testing or regular shadow link management, see xref:deploy:redpanda/manual/resilience/shadowing.adoc[]. Ensure you have completed the xref:deploy:redpanda/manual/resilience/shadowing.adoc#disaster-readiness-checklist[disaster readiness checklist] before an emergency occurs.
+====
+
+== Emergency failover procedure
+
+Follow these steps in order during an active disaster:
+
+1. <<assess-situation,Assess the situation>>
+2. <<verify-shadow-status,Verify shadow cluster status>>  
+3. <<document-state,Document current state>>
+4. <<initiate-failover,Initiate failover>>
+5. <<monitor-progress,Monitor failover progress>>
+6. <<update-applications,Update application configuration>>
+7. <<verify-functionality,Verify application functionality>>
+8. <<cleanup-stabilize,Clean up and stabilize>>
+
+[[assess-situation]]
+=== Assess the situation
+
+Confirm that failover is necessary:
+
+[,bash]
+----
+# Check if the primary cluster is responding
+rpk cluster info --brokers <primary-cluster-brokers>
+
+# If primary cluster is down, check shadow cluster health
+rpk cluster info --brokers <shadow-cluster-brokers>
+----
+
+**Decision point**: If the primary cluster is responsive, consider whether failover is actually needed. Partial outages may not require full disaster recovery.
```

Comment:
Can you offer additional guidance re what type of partial outages require full DR and which would not? Perhaps list a couple of examples that illustrate each?